examples/quickstart.ipynb

```diff
@@ -238,7 +238,7 @@
         "\n",
         "import os\n",
         "os.chdir('/content')\n",
-        "!git clone https://github.com/OpenMined/PipelineDP.git\n",
+        "!pip install pipeline-dp\n",
```

Comment:
Optional: We could also leave out the commented version of the cloning and say something like:

# If you'd like to test out the most recent version of PipelineDP, 
# uncomment the following line and comment out the previous line.
# !git clone https://github.com/OpenMined/PipelineDP.git

(no strong opinion, just a thought if someone (or ourselves) want to test colab with the HEAD version)

Reply:
Done. Thanks!

Reply:
Oh, I did it for the advanced colab only to keep quickstart simple.